src/lib/object-class.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Returns the object class for any value.
+ *
+ * @param {*} [value] - A value whose object class should be determined.
+ *
+ * @example
+ * const isNull = objectClass(null) === 'null'
+ * const isPOJO = objectClass({}) === 'object'
+ *
+ * @returns {String}
+ */
+const objectClass = (value) => /\[\w+ (\w+)]/.exec(Object.prototype.toString.call(value))[1].toLowerCase()
```

Comment:
Fair point, my reservation with it is the fact that we're not dealing entirely with what are considered, from a developer's intuitions, objects. Further, we do not care what _class_ of objects things are, which is primarily what `objectClass` seems to be designed to solve. We have a bunch of primitives that, while inheriting from a prototype and thus technically objects, are not actually _instances_ of a class. With this, I have a hard time resolving asking what type something is while at the same time implying, through naming, that it is a specific _class_ of object.

```js
2 instanceof Number             // => false
new Number(2) instanceof Number // => true
```

Most of the examples on that MDN page deal with figuring out what _sort_ of object something is; i.e., that an instance of `Dog` is more than just a plain object.

I think we both understand the JS fundamentals this conversation is centered around, so I'm going to save us any further time investment and headache involved in bikeshedding a function name, but I wanted to clarify my position. Your call, I will ultimately use whatever you decide is best.